Deployment of an assign.mutations.gatekeeper.sh object causes panic

**What steps did you take and what happened:**
[A clear and concise description of what the bug is.]
Deploy an assign mutation object causes this error: `panic: assignment to entry in nil map`. Built from https://github.com/open-policy-agent/gatekeeper/commit/6ceb3f486a0a4b4887526e982762e6857829843a

**What did you expect to happen:**


**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

```
panic: assignment to entry in nil map [recovered]
	panic: assignment to entry in nil map

goroutine 495 [running]:
k8s.io/apimachinery/pkg/util/runtime.HandleCrash(0x0, 0x0, 0x0)
	/go/src/github.com/open-policy-agent/gatekeeper/vendor/k8s.io/apimachinery/pkg/util/runtime/runtime.go:55 +0x10c
panic(0x1a680a0, 0x1ebc780)
	/usr/local/go/src/runtime/panic.go:969 +0x1b9
github.com/open-policy-agent/gatekeeper/pkg/mutation/schema.(*DB).upsert(0xc000c49e00, 0x7fb6d1cb5150, 0xc00040d500, 0xc000fab901, 0x40b82a, 0x1b0fbc0)
	/go/src/github.com/open-policy-agent/gatekeeper/pkg/mutation/schema/schema.go:103 +0x6a7
github.com/open-policy-agent/gatekeeper/pkg/mutation/schema.(*DB).Upsert(0xc000c49e00, 0x7fb6d1cb5150, 0xc00040d500, 0x0, 0x0)
	/go/src/github.com/open-policy-agent/gatekeeper/pkg/mutation/schema/schema.go:85 +0x85
github.com/open-policy-agent/gatekeeper/pkg/mutation.(*System).Upsert(0xc000c49e00, 0x1f312a0, 0xc00040d490, 0x0, 0x0)
	/go/src/github.com/open-policy-agent/gatekeeper/pkg/mutation/system.go:48 +0x885
github.com/open-policy-agent/gatekeeper/pkg/controller/assign.(*Reconciler).Reconcile(0xc000b2c0c0, 0x0, 0x0, 0xc000231940, 0xd, 0xc000ef5b20, 0xc000f95050, 0xc000f94fc8, 0xc000f94fc0)
	/go/src/github.com/open-policy-agent/gatekeeper/pkg/controller/assign/assign_controller.go:146 +0x485
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler(0xc000b27ef0, 0x1adefc0, 0xc000ef5a20, 0x0)
```

**Environment:**

- Gatekeeper version:
- Kubernetes version: (use `kubectl version`):

--- a/pkg/mutation/system.go
+++ b/pkg/mutation/system.go
@@ -25,6 +25,7 @@ type System struct {
 // NewSystem initializes an empty mutation system
 func NewSystem() *System {
 	return &System{
+		schemaDB:        *schema.New(),
 		orderedMutators: make([]types.Mutator, 0),
 		mutatorsMap:     make(map[types.ID]types.Mutator),
 	}

